Jenkinsfile

```diff
@@ -196,12 +230,56 @@ pipeline {
                             archiveArtifacts artifacts: '**/*.log'
                         }
                         cleanup {
-                        dir('test/output') {
-                            deleteDir()
+                            dir('test/output-centos') {
+                                deleteDir()
+                            }
                         }
                     }
                 }
-                }
+                stage('Single node cart-master') {
+                    agent {
+                        label 'ci_vm1'
+                    }
+                    options {
+                        timeout(time: 60, unit: 'MINUTES')
+                    }
+                    steps {
+                        provisionNodes NODELIST: env.NODELIST,
+                           node_count: 1,
+                           snapshot: true
+                        runTest stashes: [ 'CentOS-master-install', 'CentOS-master-build-vars' ],
+                                script: """set -x
+                                    . ./.build_vars-Linux.sh
+                                    CART_BASE=\${SL_PREFIX%/install*}
```

Comment:
this is non-blocking, but i would prefer this to be changed later to IOF_BASE instead of using CART_